street_network/src/initial.rs

```diff
@@ -56,6 +58,7 @@ pub struct Intersection {
     pub polygon: Polygon,
     pub roads: BTreeSet<OriginalRoad>,
     pub complexity: IntersectionComplexity,
+    pub conflict_level: ConflictType,
```

Comment:
I only add it because i need to reference it when "rendering" to geojson. Maybe i should be more creative about chasing down the value for that use case